content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,56 @@ positional arguments:
 `learning-rate`, `batch_size`, etc.
 
-In contrast to a regular <abbr>dependency</abbr>, a parameter is not a file (or
-directory). Instead, it consists of a _parameter name_ (or key) to find inside a
-YAML 1.2, JSON, TOML, or [Python](#examples-python-parameters-file) _parameters
-file_. Multiple parameter dependencies can be specified from one or more
-parameters files.
-
-The default parameters file name is `params.yaml`. Parameters should be
-organized as a tree hierarchy inside, as DVC will locate param names by their
-tree path. Parameters files have to be manually written, or generated, and these
-can be versioned directly with Git.
+To start tracking parameters, list them under the `params` field of `dvc.yaml`
+stages (manually or with the the `-p`/`--params` option of `dvc run`). For
+example:
 
-Supported parameter _value_ types are: string, integer, float, and arrays. DVC
-itself does not ascribe any specific meaning for these values. They are
-user-defined, and serve as a way to generalize and parametrize an machine
-learning algorithms or data processing code.
+```yaml
+stages:
+  preprocess:
+    cmd: ./prepare.py
+    params:
+      - html_tags
+      - cleanup.stemming
+      - myparams.toml:
+          - nlp.lemmatize
+```
 
-DVC saves the param names and their latest values in the `dvc.yaml` file. These
-values will be compared to the ones in the params files to determine if the
-stage is invalidated upon pipeline [reproduction](/doc/command-reference/repro).
+In contrast to a regular <abbr>dependency</abbr>, a parameter is not a file or
+directory. Instead, it consists of a _parameter name_ (or key) in a _parameters
+file_, where the _parameter value_ should be found. This allows you to define
+[stage](/doc/command-reference/run) dependencies more granularly: changes to
+other parts of the params file will not affect the stage. Parameter dependencies
+also prevent situations where several stages share a regular dependency (e.g. a
+config file), and any change in it invalidates all these stages, causing
+unnecessary re-executions upon `dvc repro`.
+
+The default **parameters file** name is `params.yaml`, but any other YAML 1.2,
+JSON, TOML, or [Python](#examples-python-parameters-file) files can be used
+additionally (listed under `params:` with a sub-list of param values, as shown
+in the sample above) . These files are typically written manually (or they can
+be generated) and they can be versioned directly with Git.
+
+**Parameter values** should be organized in tree-like hierarchies (dictionaries)
+inside param files (see [Examples](#examples)). DVC will interpret param names
+as the tree path to find those values. Supported types are: string, integer,
+float, and arrays (groups of params). Note that DVC does not ascribe any
+specific meaning to these values.
+
+DVC saves parameter names and values in the project's
+[DVC files](/doc/user-guide/dvc-files) in order to track them over time. They
```

Comment:
in this case we can be specific? `dvc.lock`?

Reply:
Agree. Specified.